src/components/common/ShareStructure/ShareStructures.vue

```diff
@@ -53,6 +53,7 @@ export default class ShareStructures extends Mixins(CommonMixin) {
   // Global getters
   @Getter getComponentValidate!: boolean
   @Getter getNewResolutionDates!: string[]
+  // *** TODO: refactor shared component to use original shares instead of original IA
```

Comment:
I still have work to do in Edit UI, but I'm hoping this PR can be merged so you guys can rebase and work on top of what I've already done.